dss/config.py

```diff
@@ -13,6 +13,8 @@
 from google.cloud.storage import Client
 from google.auth.transport.requests import AuthorizedSession
 from google.oauth2 import service_account
+from requests.adapters import HTTPAdapter, DEFAULT_POOLSIZE
```

Comment:
keeping these imports local reduces the spin up of our lambdas. If lambda start up becomes an issue we should start localizing imports again.